puzzles/solutions/2022/d02/p1.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+import shapes
+
+
+def get_moves(input_text: str) -> list[tuple[str, str]]:
+    """
+    :param input_text: puzzle input
+    :return: list of (opponent move, our move) couples
```

Comment:
Use the term "pair" instead of "couple".

Reply:
Fixed in 583c9232.

Reply:
Do the same for all other places "couple" is used.

Reply:
Did I miss any in that day's solutions?